index.css

```diff
@@ -14,4 +24,55 @@
 .logo:hover {
   transform: scale(1.2);
   -webkit-text-stroke: 0.2px rgba(10, 78, 10, 0.664);
-}
\ No newline at end of file
+}
+
+.header {
+  align-items: center;
+  background-color: rgba(216, 143, 143, 0.5);
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100vw;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.nav-links::after {
+  content: "";
+  background-color: rgba(34, 49, 12, 0.404);
+  display: flex;
+  flex-direction: row;
+  height: 42px;
+  position: relative;
+  right: 20px;
+  width: 2px;
+}
+
+.nav-links a {
+  background-color: transparent;
+  border: 0;
+  color: rgba(34, 49, 12, 0.404);
+  font-size: 30px;
+  margin-right: 40px;
+  transition: transform 1s;
+  text-decoration: none;
+}
+.nav-links:hover a {
+  transform: scale(1.1);
+  -webkit-text-stroke: 0.2px rgba(10, 78, 10, 0.664);
+}
```

Comment:
To be consistent with the rest of the code, please leave spaces before and after this piece of code.